Gemfile

```diff
@@ -94,6 +95,7 @@ end
 
 group :development do
   gem 'better_errors'
+  gem 'binding_of_caller'
```

Comment:
We should remove this gem it's a debug utility, not intended for production. https://rubygems.org/gems/binding_of_caller/versions/1.0.0?locale=en